.gitignore

```diff
@@ -1,65 +0,0 @@
-Thumbs.db
-
-# ignore jenkins build info
-/build.prop
-
-# package-lock.json
-package-lock.json
-
-# ignore node_modules created by grunt, but not more deeply-nested node_modules
-/node_modules
-/npm-debug.log
-
-# ignore node_modules inside src
-/src/node_modules
-/src/JSUtils/node_modules
-/src/JSUtils/node/node_modules
-
-# ignore files copied from node_modules to src/thirdparty
-/src/thirdparty/CodeMirror
-/src/thirdparty/less.min.js
-/src/thirdparty/preact
-/src/thirdparty/preact-compat
-/src/thirdparty/preact-test-utils
-/src/thirdparty/simulate-event
-/src/thirdparty/xtend
-/src/thirdparty/acorn
-
-# ignore compiled files
-/dist
-/src/.index.html
-/src/styles/brackets.min.css
-/src/styles/brackets.min.css.map
-
-# ignore everything in the dev extension directory EXCEPT the README
-# (so that the directory is non-empty and can be in git)
-/src/extensions/dev/*
-!/src/extensions/dev/README.*
-/src/extensions/default/brackets-eslint
-
-/src/extensions/disabled
-
-# ignore .disabled file for default extensions
-/src/extensions/default/*/.disabled
-
-# generate through grunt
-/src/config.json
-
-#OSX .DS_Store files
-.DS_Store
-
-# unit test working directory
-/test/results
-/test/temp
-
-# Netbeans
-/nbproject
-
-# PhpStorm
-.idea
-
-# Files that can be automatically downloaded that we don't want to ship with our builds
-/src/extensibility/node/node_modules/request/tests/
-
-# Files build by scripts
-/src/thirdparty/semver.browser.js
```

Comment:
Is this file supposed to be removed?

Reply:
I am admittedly still somewhat new here. Could I ask you if ignoring a file is the same as removing one from a commit? I at least really think I remember someone telling me in my first few commits a while ago to ignore .gitignore, so I had tried using 'git remove <file>' to exclude it from the commit. I'd not been able to get confirmation that it was the right approach, and am still uncertain of what to do only if you have a moment to help me clarify, please.